src/napari_threedee/annotators/dipoles/annotator.py

```diff
@@ -155,6 +155,10 @@ def _set_direction_from_mouse_event(self, event: Event = None):
         )
         dipole_position_3d = self.active_dipole_position[displayed_dims]
         direction = dipole_north_position_3d - dipole_position_3d
+        norm = np.linalg.norm(direction)
+        if norm == 0:
+            self.viewer.status = 'vector norm = 0'
```

Comment:
do you think users are expecting something to happen and would be surprised if this message didn't show up? Otherwise let's proceed silently instead 🙂